src/state/panel-title-mixin.ts

```diff
@@ -3,7 +3,7 @@ import { Constructor, HomeAssistant } from "../types";
 import { HassBaseEl } from "./hass-base-mixin";
 
 const setTitle = (title: string | undefined) => {
-  document.title = title ? `${title} - Home Assistant` : "Home Assistant";
+  document.title = title ? `${title} – Home Assistant` : "Home Assistant";
```

Comment:
That is one place where I am not too adamant about using a dash since most other websites seem to use a hyphen. So would be fine with reverting that one.